applications: nrf5340_audio: dfu clean up (#9275)
nrfconnect/sdk-nrf: applications/nrf5340_audio/README.rst

The generated FOTA build files use the following naming patterns:

> * For multi-image DFU, the file name refers to ``dfu_application.zip``.
>   This file updates two cores with one single file.

Reviewer: ```suggestion
* For multi-image DFU, the file is called ``dfu_application.zip``.
  This file updates two cores with one single file.
```

> * For single-image DFU, the bin file for the application core refers in the name to ``app_update.bin``.

Reviewer 2: Should we even mention single image DFU?

Author: > Should we even mention single image DFU?

I think so, we still have internal layout option.

Reviewer 2: Could we then have a disclaimer here as well? Or link to the documentation? We really do not want to use this approach.

Author: We have already stated that in caution part.
https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-9275/nrf/applications/nrf5340_audio/README.html#configuring-fota-upgrades

> * For single-image DFU, the bin file for the application core refers in the name to ``app_update.bin``.

Reviewer: ```suggestion
* For single-image DFU, the bin file for the application core is called ``app_update.bin``.
```

>   The bin file for the network core refers in the name to ``net_core_app_update.bin``.

Reviewer: ```suggestion
  The bin file for the network core is called ``net_core_app_update.bin``.
```

>   In this scenario, the cores are updated one by one with two separate files in two actions.
> 

Reviewer: Add one extra line after this list:
```suggestion
  In this scenario, the cores are updated one by one with two separate files in two actions.

See :ref:`app_build_output_files` for more information about the image files.

```

.. |net_core_hex_note| replace:: The network core for both gateway and headsets is programmed with the precompiled Bluetooth Low Energy Controller binary file :file:`ble5-ctr-rpmsg_<XYZ>.hex`, where *<XYZ>* corresponds to the controller version, for example :file:`ble5-ctr-rpmsg_3216.hex`.
   This file includes the LE Audio Controller Subsystem for nRF53 and is provided in the :file:`applications/nrf5340_audio/bin` directory.

>    If :ref:`DFU <nrf53_audio_app_configuration_configure_fota>` is enabled, the subsystem's binary file will be generate in :file:`build/zephyr/` folder (:file:`net_core_app_signed.hex`).

Reviewer: ```suggestion
   If :ref:`DFU is enabled <nrf53_audio_app_configuration_configure_fota>`, the subsystem's binary file will be generated in the :file:`build/zephyr/` directory and will be called :file:`net_core_app_signed.hex`.
```

Reviewer: @tejlmand , net_core_app_signed.hex is not listed as one of the output files here: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/app_build_system.html#output-build-files 

Should we add it to one of the tables?
